OWML.ModHelper/ModBehaviour.cs

```diff
@@ -28,5 +31,7 @@ public IList<IModBehaviour> GetDependencies()
         {
             return ModHelper.Interaction.GetDependencies(ModHelper.Manifest.UniqueName);
         }
+
+        public virtual object GetApi() => null;
```

Comment:
Huh? :P

Reply:
this gets called to get the mod API. a mod would override that method to return the API class

Reply:
basically the same as the Configure method, except it returns the interface

Reply:
Ah, cool. Could you make it look like Configure (empty method with curly brackets), and put them together? Or doesn't it work then?

Reply:
the method needs to returns something, so I guess I could just put a return null in the brackets to make it more like configure

Reply:
Ah, whatever. Just put the virtual stuff together :)